src/imitation/scripts/train_rl.py

```diff
@@ -102,6 +105,10 @@ def train_rl(
         # so normalizing it makes training more stable. Note we do *not* normalize
         # observations here; use the `NormalizeFeaturesExtractor` instead.
         venv = VecNormalize(venv, norm_obs=False, **normalize_kwargs)
+        if reward_type == "RewardNet_normalized":
```

Comment:
Maybe we should ban the behavior of double normalization. This could produce silent error if users missed the warning. I would set an Assertion error check here.

Reply:
Sure I can make it an assertion error.